hudi-common/src/main/java/org/apache/hudi/common/table/log/AbstractHoodieLogRecordReader.java

```diff
@@ -253,6 +280,13 @@ private void scanInternalV1(Option<KeySpec> keySpecOpt) {
         final String instantTime = logBlock.getLogBlockHeader().get(INSTANT_TIME);
         totalLogBlocks.incrementAndGet();
         if (logBlock.isDataOrDeleteBlock()) {
+          if (this.tableVersion.lesserThan(HoodieTableVersion.EIGHT)) {
```

Comment:
we also need to skip/handle CORRUPT_BLOCK.. For HDFS, there will be partially written log files, that fail in between.

Reply:
Maybe I lost something, but we didn't remove any corrupt log block handling in 1.x.

Reply:
yes .. its in tact..